client/src/components/PastaShow.js

```diff
@@ -28,16 +35,67 @@ const PastaShow = (props) => {
   useEffect(() => {
         getPasta()
     }, [])
-  
+
+    const postReview = async (newReviewData) => {
+        try {
+            const response = await fetch (`/api/v1/pastas/${pastaId}/review`, {
```

Comment:
Pasta has many `reviews`, we should follow proper RESTful pathing `api/v1/pastas/:id/reviews`
https://restfulapi.net/http-methods/